src/test/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationServiceTest.java

```diff
@@ -918,6 +1050,57 @@ void shouldFilterOutCollectionItemsHigherClassification() throws IOException {
             );
         }
 
+        @Test
+        // TODO Target implementation, see RDM-1204
```

Comment:
New tests that were missing for SecurityClassificationService to increase branch coverage.